pandunia-lekse.csv

```diff
@@ -2178,7 +2207,10 @@ radium|mate|Ra||radium||radio|||||||||||||radiumo|radium|rad
 radometer||||radiometer||radiómetro||||||||||||||säteilymittari|radoiometr
 radon|mate|Rn||radon||radón|||||||||||||radono|radon|radon
 raite|||eng:right|right hand side||lado derecho||||||||||||||oikea puoli|prawa strona
+raiti||||right hand||derecho|||||||||||||||
 raitiste||||rightist (right-winger)||derechista||||||||||||||oikeistolainen|prawicowiec (prawak)
+raito||||rightward||a la derecha|||||||||||||||
+raitolefti||||horizontal (lateral)||horizontal|||||||||||||||
```

Comment:
no malo... me fikra mome abla fata lekse "horizone" e "horizoni".

Reply:
me fikra kombo lekse max bon, putongo.

Reply:
a raite e lefte no si horizoni. lole si dixe da bode.

Reply:
me gara te sahi. me fikra ko fiziker; pa me si 3 dixe: gawotali (vertical), raitolefti (horizontal), e predobadi (fordinal). a pa normi ren, we da logu te.